### PR TITLE
Convert Avatar & Badge components to Composition API

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

### CODE REVIEW

File: components/ui/Badge.vue [hunk 1]
```diff
@@ -49,33 +49,16 @@ import LockIcon from '~/assets/images/utils/lock.svg'
 import CalendarIcon from '~/assets/images/utils/calendar.svg'
 import CloseIcon from '~/assets/images/utils/check-circle.svg'
```
Comment (triphora) [inline components/ui/Badge.vue:50]: Do you think you could migrate these to omorphia icons?

**Review by triphora — COMMENTED** [review]
Looks great, just one thing

File: components/ui/Badge.vue (hunk 1 above)
Comment (Mysterious-Dev) [reply components/ui/Badge.vue:50]: All icons ?

Comment (triphora) [reply components/ui/Badge.vue:50]: In this file yeah. Whenever I touch any file, I tend to also migrate stuff there to omorphia whenever I can

Comment (Mysterious-Dev) [reply components/ui/Badge.vue:50]: I can look to do that ^^

Comment (Mysterious-Dev) [reply components/ui/Badge.vue:50]: But it requires an omorphia PR (some icons are missing).

Comment (triphora) [reply components/ui/Badge.vue:50]: Don't worry about it then!

**Review by triphora — APPROVED** [review]

**Comment by Mysterious-Dev** [comment]
> Don't worry about it then!

Okay, however I might one day make a PR for it ^^

**Pull request merged**